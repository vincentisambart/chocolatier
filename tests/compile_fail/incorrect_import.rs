use chocolatier::chocolatier;

#[chocolatier]
mod inline {
    import!(framwork = "NSFoundation");
}

fn main() {}
