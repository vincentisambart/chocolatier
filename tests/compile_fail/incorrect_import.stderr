error: expected `framework`
 --> $DIR/incorrect_import.rs:5:13
  |
5 |     import!(framwork = "NSFoundation");
  |             ^^^^^^^^
